### DISCUSSION TITLE
Using .Rmd files in Quarto?

### DISCUSSION THREAD

**Discussion by aloboa** [quarto-dev/quarto-cli#3276] [Q&A]
I am starting to use Quarto to create a website. I have quite a number or R notebooks in .Rmd and .nb.html formats. Is there a way I can include these .Rmd files in the Quarto-generated website (eg as posts) or should I copy-paste and adapt each.Rmd file into a .qmd file? An obvious solution would be to make a new .qmd file for each post with just a link to each .nb.html file, but this is not nice.

**Comment by jlgraves-ubc** [comment]
I don't think you need to do anything.  You can just include `Rmd` files directly in a Quarto website without anything special.  The `qmd` format is kind of a superset of `Rmd`.

You can also use a tool like [jupytext](https://jupytext.readthedocs.io/en/latest/) if you have a bunch of formats to convert (no copy pasting!); it adds nice metadata to the `qmd` format, in my experience.
